lib/openshift/report.rb

```diff
@@ -21,7 +21,7 @@ def conditions(user: nil, cached: false, quiet: false)
 
     def running?(user: nil, cached: false, quiet: false)
       # the reason 'Scheduled' is an initial state what we
-      conditions.any? { |c| c.type == 'Running' and c.reason != 'Scheduled' }
+      conditions.any? { |c| c.type == 'Running' }
```

Comment:
shold this check that the status is `true` or something? Conditions can be true or false, no?

Reply:
If this is not the case, then you can merge without me if that's blocking you.

Reply:
@akostadinov, this is correct, we are just checking the word `Running` is there.